File: Geolocator-master/app/src/main/java/com/sample/geolocator/demo/java/GeofenceIntentService.java
package com.sample.geolocator.demo.java;

import android.util.Log;

import com.sample.geolocator.demo.misc.UtilsKt;
import com.sample.geolocator.geofencer.models.Geofence;

import org.jetbrains.annotations.NotNull;

public class GeofenceIntentService extends com.sample.geolocator.geofencer.service.GeofenceIntentService {

    @Override
    public void onGeofence(@NotNull Geofence geofence) {

        Log.v(GeofenceIntentService.class.getSimpleName(), "onGeofence " + geofence);

        UtilsKt.sendNotification(
                getApplicationContext(),
                geofence.getTitle(),
                geofence.getMessage()
        );
    }
}

File: Geolocator-master/app/src/main/java/com/sample/geolocator/demo/kotlin/GeofenceIntentService.kt
package com.sample.geolocator.demo.kotlin

import android.util.Log
import com.sample.geolocator.demo.misc.sendNotification
import com.sample.geolocator.geofencer.models.Geofence


class GeofenceIntentService : com.sample.geolocator.geofencer.service.GeofenceIntentService() {

    override fun onGeofence(geofence: Geofence) {
        Log.v(com.sample.geolocator.demo.java.GeofenceIntentService::class.java.simpleName, "onGeofence $geofence")
        sendNotification(
            applicationContext,
            geofence.title,
            geofence.message
        )
    }
}